libraries/app/include/graphene/app/api.hpp

```diff
@@ -185,31 +185,29 @@ namespace graphene { namespace app {
                                                                         unsigned limit = 100,
                                                                         uint32_t start = 0) const;
          /**
-          * @brief Get history of orders with b:a pair
+          * @brief Get list of orders with b:a pair
```

Comment:
Before you change the code, can you please explain why you think it's better to change this way? Is there other/better alternatives? Have you understood the purpose of the API? To me neither "history of orders" nor "list of orders" fits this API. Actually I think the word "orders" is not the best here.

Reply:
maybe it's best fitted here to use "asset pairs" instead of "history/list of orders", because looking at code realization here https://github.com/bitshares/bitshares-core/blob/develop/libraries/app/api.cpp#L286-L309 I see that result is constructed with asset pairs a:b.
Am I wrong here or not ? Maybe I've missed something here ?

Reply:
I think you still didn't get it. It's not your fault, because we lack many docs about the code here and elsewhere.

Please ask a trader how to describe/explain the API better.

>because looking at code ...

Please look deeper.

Reply:
OK. Sure. Thanks !